src/main/java/org/jitsi/jigasi/TranscriptionGatewaySession.java

```diff
@@ -550,6 +551,14 @@ public String getMucDisplayName()
         return TranscriptionGatewaySession.DISPLAY_NAME;
     }
 
+    @Override
+    public void notify(Transcriber transcriber, TranscriptEvent event) {
+        TranscriptionStatusExtension extension = new TranscriptionStatusExtension();
+        extension.setStatus( transcriber.isTranscribing()
+                ? TranscriptionStatusExtension.Status.ON : TranscriptionStatusExtension.Status.OFF);
+        jvbConference.sendPresenceExtension(extension);
```

Comment:
It looks like this will update its presence with every new transcription result. Or am I missing something? I would expect it to update its presence when the status changes (when it starts and when it stops).

Reply:
Actually, it implements TranscriptionEventListener but isn't registered anywhere as a listener?